metric-collector/service-readiness/kubeflow-readiness.py

```diff
@@ -0,0 +1,140 @@
+import argparse
+from time import sleep, time
+
+import google.auth
+import google.auth.app_engine
+import google.auth.compute_engine.credentials
+import google.auth.iam
+from google.auth.transport.requests import Request
+import google.oauth2.credentials
+import google.oauth2.service_account
+from prometheus_client import start_http_server, Gauge
+import requests
+
+IAM_SCOPE = 'https://www.googleapis.com/auth/iam'
```

Comment:
Why do we need to hard code these URLs aren't they part of the client library?

Reply:
Surprisingly seems all user guide use hard code URL directly.